<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventAlarmForm</class>
 <widget class="QWidget" name="EventAlarmForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>规则名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="alarmRuleNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>219</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>遥测超限1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>告警原因:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alarmCauseComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>告警类型:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alarmTypeComboBox">
         <item>
          <property name="text">
           <string>事故信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>故障信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>变位信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>越限信息</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>告知信息</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="cosDisplayLabel">
         <property name="text">
          <string>变位显示:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cosDisplayComboBox">
         <item>
          <property name="text">
           <string>分开/闭合</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>动作/复归</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>分闸/合闸</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>正常/中断</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>告警子原因：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExListWidget" name="subCauseListWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>241</height>
      </size>
     </property>
     <property name="title">
      <string>选择装置或系统点</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePushButton">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExListWidget</class>
   <extends>QListWidget</extends>
   <header>exlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>alarmRuleNameLineEdit</tabstop>
  <tabstop>alarmCauseComboBox</tabstop>
  <tabstop>alarmTypeComboBox</tabstop>
  <tabstop>cosDisplayComboBox</tabstop>
  <tabstop>subCauseListWidget</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>deletePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
